web/index.html

```diff
@@ -129,7 +129,7 @@ <h1>Umiaq</h1>
           if (status === 'timed_out') {
             meta += "\n⚠️ Solver timed out. Results may be incomplete. ⚠️";
           } else if (status === 'found_enough') {
-            meta += `\nSearch terminated early.`;
+            meta += `\nSearch terminated after finding ${count} results.`;
```

Comment:
maybe explicitly note how many were requested here so it's clear that we didn't end because we gave up but ended at (or even a bit later than) _the user_ asked for